sdk/cosmos/examples/create_delete_database.rs

```diff
@@ -62,29 +65,12 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     {
         let db_client = client.clone().into_database_client(database_name.clone());
 
-        let indexes = IncludedPathIndex {
-            kind: KeyKind::Hash,
-            data_type: DataType::String,
-            precision: Some(3),
-        };
-
-        let ip = IncludedPath {
-            path: "/*".to_owned(),
-            indexes: Some(vec![indexes]),
-        };
-
-        let ip = IndexingPolicy {
-            automatic: true,
-            indexing_mode: IndexingMode::Consistent,
-            included_paths: vec![ip],
-            excluded_paths: vec![],
-        };
-
         let create_collection_response = db_client
-            .create_collection("/id")
-            .indexing_policy(ip)
-            .offer(Offer::Throughput(400))
-            .execute("panzadoro")
+            .create_collection(
+                Context::new(),
```

Comment:
Discussed this offline with @JeffreyRichter. The Context argument is actually the first argument in C++ and Go. I _personally_ am a fan of it being last, but we decided to open an issue to discuss this instead of blocking this PR.